app/Traits/RememberDeviceTrait.php

```diff
@@ -34,6 +34,8 @@ public function getRememberDeviceToken()
     public function setRememberDeviceToken($value)
     {
         if (! empty($this->getRememberDeviceTokenName())) {
+            // Make sure new value is immediately saved to db, as well as to model object.
+            $this->where('id', $this->id)->update([$this->getRememberDeviceTokenName() => $value]);
```

Comment:
Can't we just call `$this->save()` after updating the attribute? Also looking at the Laravel implementation it looks like they prevent timestamps from updating during this process for some reason?
```php
        $user->setRememberToken($token);
        $timestamps = $user->timestamps;
        $user->timestamps = false;
        $user->save();
        $user->timestamps = $timestamps;
```
https://github.com/laravel/framework/blob/6.x/src/Illuminate/Auth/EloquentUserProvider.php#L80-L98

Reply:
I suppose they want to keep the update_timestamp reflecting when the user purposefully updated their data.  That's sort of what I was trying to do here by only updating the one field.

Reply:
I'll switch it do saving the same way as laravel does the remember token there. It'll be consistent, at least.